qiita_db/analysis.py

```diff
@@ -432,7 +432,7 @@ def jobs(self):
                "analysis_id = %s".format(self._table))
         job_ids = conn_handler.execute_fetchall(sql, (self._id, ))
         if job_ids == []:
-            return None
+            return []
```

Comment:
This is a small can of worms... but can the docstring be updated to indicate what to expect when the job isn't found?

Also, if `job_ids` comes back as `[]`, then the if statement is redundant. As in, the following should still be valid:

```python
return [job_id[0] for job_id in job_ids]
```